### DISCUSSION TITLE
Please help me to add movies into list.

### DISCUSSION THREAD

**Discussion by mailtoy** [NAIST-SE/AttractingContributors#42] [General]
URL - https://github.com/mailtoy/AttractingContributors-Movies

**Comment by noppadol-assava** [comment]
I submit a pull request to you, pls review it.

**Reply by mailtoy** [reply]
Thank you Tor-san!

**Comment by tattiyaS** [comment]
I add a movie yo your list. yay!

**Reply by mailtoy** [reply]
I saw it. Thanks!

**Comment by vittunyuta-aom** [comment]
added!!

**Reply by mailtoy** [reply]
Good job :)
